Formula/alluxio.rb

```diff
@@ -11,13 +11,35 @@ def install
     libexec.install Dir["*"]
     bin.write_exec_script Dir["#{libexec}/bin/*"]
 
+    rm_rf Dir["#{etc}/alluxio/*"]
+
     (etc/"alluxio").install libexec/"conf/alluxio-env.sh.template" => "alluxio-env.sh"
     ln_sf "#{etc}/alluxio/alluxio-env.sh", "#{libexec}/conf/alluxio-env.sh"
+    (etc/"alluxio/alluxio-site.properties").write <<~EOS
```

Comment:
The default configuration should be provided (and maintained) in the upstream project, and installed as part of the default installation method. Or, at the very least, provided as an example file (`alluxio-site.properties.example`) which we can then rename and install as configuration file.

But we do not want to maintain the content of the software's configuration. If you, as the upstream project, decide to ship a product that cannot run without configuration, then that's your choice.

Reply:
@fxcoudert Can we do the same thing Zookeeper does? https://github.com/Homebrew/homebrew-core/blob/master/Formula/zookeeper.rb#L39

```rb
  def default_alluxio_conf
    <<~EOS
      alluxio.master.hostname=localhost
    EOS
  end

  def install
    ...
    defaults = etc/"alluxio/alluxio-site.properties"
    defaults.write(default_alluxio_conf) unless defaults.exist?
```

We just need to set this single property

Reply:
That would be slightly better.

Reply:
@fxcoudert better enough that we can fix this formula for people who want to use it? We have no intention to add more than just the single line of required configuration, so it really doesn't make sense to have a dedicated file just to hold the default master hostname value for running on a single node.

Reply:
OK then.